projects/plugins/jetpack/_inc/client/components/popover/index.jsx

```diff
@@ -78,10 +81,10 @@ class Popover extends Component {
 
 	UNSAFE_componentWillReceiveProps( nextProps ) {
 		// update context (target) reference into a property
-		if ( ! isDOMElement( nextProps.context ) ) {
-			this.domContext = ReactDom.findDOMNode( nextProps.context );
+		if ( ! isDOMElement( nextProps.context ) && nextProps.context !== null ) {
```

Comment:
Technically the null check isn't needed, (`isDOMElement` is `isElement` in `util.js`, and this will return false for null), but it makes it explicit and makes sure the null check will always be considered.